libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -932,6 +932,32 @@ void AC_PosControl::write_log()
                        double(accel_target.y * 0.01f),
                        double(accel_x * 0.01f),
                        double(accel_y * 0.01f));
+
```

Comment:
be nicer for both to use structure interface

Reply:
done